Use fully qualified class names for modTemplateVar

JoshuaLuckers opened the pull request:
### What does it do?
Use the fully qualified class names for modTemplateVar when possible

### Why is it needed?
To make sure we have the right class names.

### Related issue(s)/PR(s)
Fixes #14714

JoshuaLuckers committed:
Use fully qualified class names for modTemplateVar

Ruslan-Aleev reviewed (approved):
It works, thanks :)

opengeek commented on core/src/Revolution/Sources/mysql/modMediaSourceElement.php:
```diff
@@ -30,16 +31,16 @@ class modMediaSourceElement extends \MODX\Revolution\Sources\modMediaSourceEleme
                 'phptype' => 'string',
                 'null' => false,
-                'default' => 'modTemplateVar',
+                'default' => 'MODX\\Revolution\\modTemplateVar',
```
This change needs to be made in the schema files and updated by using the xpdo parse-schema command.

JoshuaLuckers replied:
@opengeek im not sure what to change in the schema file to reflect this change, maybe I’m overlooking something.

Mark-H replied:
https://github.com/modxcms/revolution/blob/3.x/core/model/schema/modx.sources.mysql.schema.xml#L69

opengeek replied:
I'll get this resolved today and post the command so we have it for future reference when updating the model.

opengeek replied:
Here were the commands to update the files once the schemas were updated:

```
core/vendor/bin/xpdo parse-schema --config=core/model/schema/config.php --update=1 --psr4=MODX\\ mysql core/model/schema/modx.sources.mysql.schema.xml core/src/
core/vendor/bin/xpdo parse-schema --config=core/model/schema/config.php --update=1 --psr4=MODX\\ sqlsrv core/model/schema/modx.sources.sqlsrv.schema.xml core/src/
```

Let me know if you have any questions.

JoshuaLuckers replied:
Done, thanks @Mark-H and @opengeek

opengeek committed:
Update platform classes via changes to the model schema

After making changes to the schema, these commands were executed to update the maps in the platform classes:

```
core/vendor/bin/xpdo parse-schema --config=core/model/schema/config.php --update=1 --psr4=MODX\\ mysql core/model/schema/modx.sources.mysql.schema.xml core/src/
core/vendor/bin/xpdo parse-schema --config=core/model/schema/config.php --update=1 --psr4=MODX\\ sqlsrv core/model/schema/modx.sources.sqlsrv.schema.xml core/src/
```
